automatic/kubernetes-cli/tools/chocolateyInstall.ps1

```diff
@@ -5,21 +5,23 @@ $packageName = 'kubernetes-cli'
 $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
 $installLocation = "$(Get-ToolsLocation)\$packageName"
 
-$packageArgs1 = @{
+$packageArgs = @{
   PackageName    = $packageName
   FileFullPath   = Get-Item $toolsPath\*-386.tar.gz
   FileFullPath64 = Get-Item $toolsPath\*-amd64.tar.gz
   Destination    = $installLocation
 }
-Get-ChocolateyUnzip @packageArgs1
+Get-ChocolateyUnzip @packageArgs
 
-$packageArgs2 = @{
-  PackageName    = $packageName
-  FileFullPath   = Get-Item $installLocation\*-386.tar
-  FileFullPath64 = Get-Item $installLocation\*-amd64.tar
-  Destination    = $installLocation
+if (!(Test-Path "$installLocation\kubernetes\client" -include "*.exe")) {
```

Comment:
hmm, while I'm the one that asked for a test case, I think this may work against what we would want.

In cases of updating the package, this check would be true all the time.
Would also need to remove the directory before it's extracted the first time.

Could I ask you to update again?

Reply:
At the end of the installation, the script is cleaning after unpack:
```
Remove-Item "$installLocation\kubernetes" -Recurse -Force -ea 0
Remove-Item "$installLocation\*.tar"
```
This should cover updates, right?

Reply:
yeah, sorry. I just reviewed the changes and completely forgot about that